notifications.go

```diff
@@ -405,19 +405,37 @@ func (api *API) GetAvailableNotificationTypes(ctx context.Context, accountID str
 	return r, nil
 }
 
+// TimeRange is an object for filtering the alert history based on timestamp.
+type TimeRange struct {
+	Since  string `json:"since,omitempty"`
```

Comment:
something we can do here is apply `url` tags to these fields and instead automatically `Encode` them.

example from `tunnel` - https://github.com/cloudflare/cloudflare-go/blob/fcb3cd41e72cbaeeeb952df36b69d0c4bbff1775/tunnel.go#L105-L113

I might even drop this onto `PaginationOptions` too. WDTY?